python/pyspark/ml/clustering.py

```diff
@@ -330,6 +357,20 @@ class KMeans(JavaEstimator, HasFeaturesCol, HasPredictionCol, HasMaxIter, HasTol
     >>> df = spark.createDataFrame(data, ["features"])
     >>> kmeans = KMeans(k=2, seed=1)
     >>> model = kmeans.fit(df)
+    >>> summary = model.summary
+    >>> summary.k
+    2
+    >>> summary.predictionCol
```

Comment:
things like this are better left for the `tests.py` file, since this shows up in the doc and might not be useful to users